tagging.py

```
@@ -1,16 +1,503 @@
+        return f"{ self.val }: index: { self.pos }: { self.typ }"
+    
+class LexerFunc( Protocol ):
+    
+    def __call__( self, __origin: 'Lexer') -> 'LexerFunc | None': ... # type: ignore
+    
+
+
+
+class Lexer:
+    
+    def __init__( self, string: str, allow_issue_start_with_letter: bool = False) -> None:
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)